lib/make-middleware.js

```diff
@@ -94,8 +94,10 @@ function makeMiddleware (setup) {
 
     // handle files
     busboy.on('file', function (fieldname, fileStream, filename, encoding, mimetype) {
-      // don't attach to the files object, if there is no file
-      if (!filename) return fileStream.resume()
+      // filename is not required (https://tools.ietf.org/html/rfc1867) but if
+      // filename not present busboy only treats as file if content type is
+      // application/octet-stream
+      if (!filename) filename = 'undefined'
```

Comment:
What happens if we get multiple files with no filename? Is having two files set to `filename == 'undefined'` an issue in anyway? Maybe not if the latest version returns an array rather than a key/value object?

Reply:
it is not used for anything else internally afaict. the single file case is the only place where multiple files would overwrite and that is based on fieldname.